docs/porting/overview.md

```diff
@@ -1,5 +1,68 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+If you're coming from .NET Framework you may be interested in running your code on .NET Core.  Before any process is outlined, it's helpful to take a step back and evaluate your reasons for wanting to port code to run on .NET Core.
+
+## Should you port?
```

Comment:
I'll add this before the "good candidate/less than ideal candidate" sections.  Good point!